src/test/groovy/hudson/plugins/batch_task/BatchRunTest.groovy

```diff
@@ -1,26 +0,0 @@
-package hudson.plugins.batch_task
-
-import org.jvnet.hudson.test.HudsonTestCase
-
-/**
- *
- * @author Kohsuke Kawaguchi
- */
-public class BatchRunTest extends HudsonTestCase {
```

Comment:
Because all the tests were on an old-legacy status and some tests were failing - so decided to update everything.